subsys/net/lib/lwm2m/lwm2m_senml_cbor_encode.c

```diff
@@ -37,9 +36,8 @@ static bool encode_repeated_record_bn(zcbor_state_t *state, const struct record_
 	bool tmp_result = ((((zcbor_int32_put(state, (-2)))) &&
 			    (zcbor_tstr_encode(state, (&(*input)._record_bn)))));
 
-	if (!tmp_result) {
```

Comment:
Is this clang rules different than normal zephyr. I was wondering that {
} is mandatory.

Reply:
I think the clang-format automatically uses the zephyr rules.